lib/env.mjs

```diff
@@ -34,7 +34,11 @@ export default createEnv({
 		LANGCHAIN_PROJECT: process.env.LANGCHAIN_PROJECT,
 		SERPAPI_API_KEY: process.env.SERPAPI_API_KEY,
 		WEAVIATE_SCHEME: process.env.WEAVIATE_SCHEME,
-		WEAVIATE_HOST: process.env.WEAVIATE_HOST
+		WEAVIATE_HOST: process.env.WEAVIATE_HOST,
+		DASHBOARD_GITHUB_ID: process.env.DASHBOARD_GITHUB_ID,
+		DASHBOARD_GITHUB_SECRET: process.env.DASHBOARD_GITHUB_SECRET,
```

Comment:
Are these different from `GITHUB_APP_ID` and `GITHUB_PRIVATE_KEY` @adamgobes?

Reply:
These point to the oauth app (login with github)

Reply:
^ yes, this is correct